src/Elcodi/Bundle/ProductBundle/Resources/config/doctrine/Pack.orm.yml

```diff
@@ -0,0 +1,186 @@
+Elcodi\Component\Product\Entity\Pack:
+    type: entity
+    repositoryClass: Elcodi\Component\Product\Repository\PackRepository
+    table: pack
+    id:
+        id:
+            type: integer
+            generator:
+                strategy: AUTO
+    fields:
+        name:
+            column: name
+            type: string
+            length: 255
+            nullable: false
+        sku:
+            column: sku
+            type: string
+            length: 255
+            nullable: true
+        slug:
+            column: slug
+            type: string
+            length: 255
+            nullable: false
+        shortDescription:
+            column: short_description
+            type: string
+            length: 255
+            nullable: true
+        description:
+            column: description
+            type: text
+            nullable: true
+        showInHome:
+            column: show_in_home
+            type: boolean
+            nullable: true
+        dimensions:
+            column: dimensions
+            type: string
+            length: 255
+            nullable: true
+        stock:
+            column: stock
+            type: integer
+            nullable: true
+        stockType:
+            column: stock_type
+            type: integer
+            nullable: false
+        price:
+            column: price
+            type: integer
+            nullable: true
+        reducedPrice:
+            column: reduced_price
+            type: integer
+            nullable: true
+        height:
+            column: height
+            type: integer
+            nullable: false
+        width:
+            column: width
+            type: integer
+            nullable: false
+        depth:
+            column: depth
+            type: integer
+            nullable: false
+        weight:
+            column: weight
+            type: integer
+            nullable: false
+        imagesSort:
+            column: images_sort
+            type: string
+            lentgh: 2048
```

Comment:
Well, this is a string, and a product could have many images. Which project do you recommend?